libs/go.json

```diff
@@ -108,6 +111,7 @@
   {"id":"go.grpc","imports":["google.golang.org/grpc"],"name":"grpc","repo":"grpc/grpc-go","tags":["microservices"],"tech":["cloud"],"status":"model-ready"},
   {"id":"go.grpc-gateway","imports":["github.com/grpc-ecosystem/grpc-gateway"],"name":"grpc-gateway","repo":"grpc-ecosystem/grpc-gateway","tags":["microservices"],"tech":["cloud"],"status":"model-ready"},
   {"id":"go.kit","imports":["github.com/go-kit/kit"],"name":"Go kit","repo":"go-kit/kit","tags":["microservices"],"tech":["cloud"],"status":"awaiting-model"},
+  {"id":"go.libcompose","imports":["github.com/docker/libcompose"],"name":"libcompose","repo":"docker/libcompose","tags":["docker-compose"],"tech":["cloud"],"status":"awaiting-model"},
```

Comment:
The docs say this lib is not maintained. I guess it does not hurt having it here, but maybe there is a live fork somewhere?

Reply:
they refer to https://github.com/docker/cli but I don't see it was used as a library